supporter-product-data/src/main/scala/com/gu/lambdas/AddSupporterRatePlanItemToQueueLambda.scala

```diff
@@ -98,60 +105,17 @@ object UpdateDynamoLambda extends StrictLogging {
       logger.info(
         s"Continuing processing with batch of ${batch.length} - time remaining: ${timeOutCheck.timeRemainingMillis / 1000} seconds, buffer: ${timeoutBufferInMillis / 1000} seconds",
       )
-
-      Await.result(writeBatch(batch, dynamoDBService, alarmService), 120.seconds)
+      try {
+        sqsService.sendBatch(batch)
+      } catch {
+        case e: Throwable =>
+          SafeLogger.error(scrub"Error sending batch to sqs\nbatch content was ${batch.asJson.spaces2}", e)
+          alarmService.triggerCsvReadAlarm
+      }
 
       val (_, highestProcessedIndex) = batch.last
       val newProcessedCount = highestProcessedIndex + 1
       logger.info(s"$newProcessedCount items processed")
       newProcessedCount
     }
-
-  def writeBatch(
-      list: List[(SupporterRatePlanItem, Int)],
-      dynamoDBService: SupporterDataDynamoService,
-      alarmService: AlarmService,
-  ) = {
-    val futures = list.map { case (supporterRatePlanItem, index) =>
-      logger.info(
-        s"Attempting to write item index $index - ${supporterRatePlanItem.productRatePlanName} to Dynamo - ${supporterRatePlanItem.asJson.noSpaces}",
-      )
-      dynamoDBService.writeItem(supporterRatePlanItem)
-    }
-    Future.sequence(futures)
-  }
-
-  def batchItemsWhichCanUpdateConcurrently(
```

Comment:
so we no longer can do anything concurrently?

Reply:
We can actually do everything concurrently, I worked through it and the ordering doesn't actually matter, although I thought  it did when I originally wrote this code.